### PR TITLE
Update inputs.jl

### PR DESCRIPTION
**Pull request by rathod-b**
Add a hybrid sizing indicator which automatically sets simulation years and max sizing iterations if its set for automatic hybrid ghx sizing

### CODE REVIEW

File: src/inputs.jl [hunk 1]
```diff
@@ -117,14 +117,16 @@ Base.@kwdef mutable struct InputsStruct
     cop_map_eft_heating_cooling::Array{Any,1} = Dict[]
     wwhp_cop_map_eft_heating::Array{Any,1} = Dict[]
     wwhp_cop_map_eft_cooling::Array{Any,1} = Dict[]
+    
     # Model Settings
-    simulation_years::Int64 = 25  # Number of years for GHP-GHX model
+    hybrid_auto_ghx_sizing_flag::Bool = false
+    simulation_years::Int64 = hybrid_auto_ghx_sizing_flag ? 2 : 25  # Number of years for GHP-GHX model
```
Comment (rathod-b) [inline src/inputs.jl]: Does this way of setting `simulation years` overwrite a user provided `simulation years`?

Comment (Bill-Becker) [reply src/inputs.jl]: Seems like it would overwrite a user provided value.

Comment (rathod-b) [reply src/inputs.jl]: Updated per your feedback.

Comment (Bill-Becker) [reply src/inputs.jl]: What did you update?

Comment (rathod-b) [reply src/inputs.jl]: Updated the input defaults so user provided inputs dont get overwritten and moved the automatic hybrid sizing logic to an if block per https://github.com/NREL/GhpGhx.jl/pull/7/commits/25184b9fea90eece740322745394146862f35613

Comment (Bill-Becker) [reply src/inputs.jl]: Ah for some reason the GitHub interface wasn't showing that change for me. That looks good.

**Review by Bill-Becker — APPROVED** [review]

**Pull request merged**